src/layouts/ultimate-guide.tsx

```diff
@@ -30,8 +31,8 @@ const UltimateGuideLayout: FunctionComponent<LayoutProps> = ({
         }}
         canonical={url}
       />
-      <div className="prose max-w-none">
-        <h1>{title}</h1>
+      <div className="prose md:prose-xl max-w-2xl mx-auto leading-6">
+        <h1 className="text-xl">{title}</h1>
```

Comment:
nitpick: h1 should move above div for its classes to take effect.

Reply:
Oh yeah didn't realise this was redundant. Removed it 72e2bac